src/markdown-pages/automate-workflows/get-started-terraform.mdx

```diff
@@ -163,147 +234,149 @@ resource "newrelic_alert_condition" "alert_condition_name" {
+</Important>
 
-    # Response time alert condition will not be provisioned unless a critical violation threshold is specified.
-    response_time_critical_threshold = 3
+```hcl
+# Slack notification channel
+resource "newrelic_alert_channel" "slack_notification" {
+  name = "slack-example"
+  type = "slack"
 
-    channel_ids = [data.newrelic_alert_channel.pager.id]
+  config {
+    # Use the URL provided in your New Relic Slack integration
+    url     = "https://hooks.slack.com/services/XXXXXXX/XXXXXXX/XXXXXXXXXX"
+    channel = "your-slack-channel-for-alerts"
+  }
 }
 ```
 
-</Step></Steps>
-
-## Tip: Avoid resources being marked as "sensitive"
-
-When API results are deemed to be secret and are obfuscated, Terraform may be unable to detect the state of a resource, marking it as changed. Consider this example:
-
-```hcl lineNumbers=false
-resource "newrelic_alert_channel" "slack" {
-  name = "slack"
-  type = "slack"
+Last, but not least, in order to apply the notification channel to your alert policy, create a [`newrelic_alert_policy_channel`](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/alert_policy_channel):
 
-  config {
-    channel = "test"
-    url     = "https://hooks.slack.com/services/xxxx/xxxxx"
-  }
+```hcl
+resource "newrelic_alert_policy_channel" "golden_signals" {
+  policy_id   = newrelic_alert_policy.golden_signal_policy.id
+  channel_ids = [newrelic_alert_channel.slack_notification.id]
 }
 ```
 
-<br />
+A `newrelic_alert_policy_channel` links the notification channel you just created to your alert policy.
 
-Running `terraform plan` yields the following:
+To finalize your golden signal alerts configuration, run `terraform apply` one last time to make sure all of your configured resources are up to date.
 
-```
-    -/+ newrelic_alert_channel.slack (new resource required)
-          id:                    "2344397" => <computed> (forces new resource)
-          config.%:              "1" => "2" (forces new resource)
-          config.channel:        <sensitive> => <sensitive> (attribute changed)
-          config.url:            <sensitive> => <sensitive> (forces new resource)
-          name:                  "slack" => "slack"
-          type:                  "slack" => "slack"
-```
+</Step>
 
-<br />
+</Steps>
 
-To prevent Terraform from wrongly marking a resource as changed, add an [`ignore_changes`](https://www.terraform.io/docs/configuration/resources.html#ignore_changes) directive:
+## Conclusion
 
-```hcl lineNumbers=false
-resource "newrelic_alert_channel" "slack" {
-  ...
-  lifecycle {
-    ignore_changes = ["config"]
-  }
-}
-  ...
-```
+As your team evaluates the alerting system you’ve put in place, you’ll find that you may need to tweak configuration values, such as the alert threshold and duration. You can do this with a pull request to your alerting repository. Your team can review these changes alongside the rest of your code contributions.
```

Comment:
This is the first time you mention a remote repository. I'd recommend spelling that out a little bit for users that might find that reference out of the blue. Maybe something like:

> If you manage your `terraform-project` in a remote repository, you can submit a pull request so your team can review these changes...